test/Microsoft.AspNet.Mvc.Core.Test/ApplicationModel/ActionModelTest.cs

```diff
@@ -78,6 +79,13 @@ public void CopyConstructor_CopiesAllProperties()
                     // Ensure non-default value
                     Assert.NotEmpty((IEnumerable<object>)value1);
                 }
+                else if (typeof(IDictionary<object, object>).IsAssignableFrom(property.PropertyType))
```

Comment:
Added the comment. https://github.com/aspnet/Mvc/blob/enhance-adp/test/Microsoft.AspNet.Mvc.Core.Test/ApplicationModel/ActionModelTest.cs#L64